pkg/kubelet/volume/secret/secret.go

```diff
@@ -71,13 +71,14 @@ func (plugin *secretPlugin) NewCleaner(volName string, podUID types.UID) (volume
 }
 
 func (plugin *secretPlugin) newCleanerInternal(volName string, podUID types.UID) (volume.Cleaner, error) {
-	return &secretVolume{volName, podUID, plugin, nil}, nil
+	return &secretVolume{volName, "", podUID, plugin, nil}, nil
 }
 
 // secretVolume handles retrieving secrets from the API server
 // and placing them into the volume on the host.
 type secretVolume struct {
 	volName   string
+	namespace string
 	podUID    types.UID
 	plugin    *secretPlugin
 	secretRef *api.ObjectReference
```

Comment:
@pmorie We're both following the same pattern here, where our volumes are referencing something in the same namespace.

Because we don't want/need the user to specify their namespace on the volume (they're already in a namespace with their pod) and I really just needed the name as a reference, @thockin suggested I dump the objectRef and just go with claimName.  

Perhaps that pattern fits here, too?